systems/gateway/.env.example

```
@@ -0,0 +1,32 @@
+# Gateway downstream service endpoints
+GATEWAY_IDENTITY_URI=http://localhost:8081
+GATEWAY_APPLICATION_URI=http://localhost:8082
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GATEWAY_APPLICATION_URI key should go before the GATEWAY_IDENTITY_URI key

(UnorderedKey)